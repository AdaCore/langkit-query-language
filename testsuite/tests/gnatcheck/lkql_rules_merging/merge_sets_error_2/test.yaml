driver: gnatcheck
description: |
  Check that defining multiple Style_Checks checks using the short format fail.
lkql_rule_file: rules3.lkql
rules_dirs:
  - set
show_rule: true
format: brief
extra_args:
  - -v
