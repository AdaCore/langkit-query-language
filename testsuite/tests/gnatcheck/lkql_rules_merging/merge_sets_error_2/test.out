rules3.lkql:1:13: info: register new instance style_checks
rules3.lkql:1:13: error: cannot add instance style_checks twice using the shortcut argument format. Previous instance has been declared in the same set
gnatcheck <version> (<date>)
Copyright <date>, AdaCore.
gnatcheck: error: no rule to check specified
try "gnatcheck --help" for more information.
>>>program returned status code 2
